content/docs/ui/account-and-settings/mail.md

```diff
@@ -57,6 +57,21 @@ Allows you to set a schedule for SendGrid to automatically delete contacts from
 
 - Hard Bounces - The number of days, after which SendGrid will purge all contacts from your hard bounces suppression lists.
 
+## Bypass
```

Comment:
I'm wondering if we should call out that using v3 mail send api provides the more granular filters in this page: https://sendgrid.com/docs/for-developers/sending-email/smtp-filters/#filter-bypasslistmanagement. though I can also see why we maybe not want to mix smtp filters up with v3 mail send api

Reply:
I think a <call-out> here linking to this granular v3 stuff could be nice. I'll add it, and it can always be removed if it's confusing.